mdwiki/index.tmpl

```diff
@@ -126,7 +138,12 @@
 	body, h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6, .tooltip, .popover {
         font-family: "Lato", "Lato Offline", "Helvetica Neue", Helvetica, Arial, sans-serif !important;
 	}
-	
+	.panel{
```

Comment:
You should fix the indentation in this added class.  The file is a little inconsistent, but let's do 4 spaces since that matches most of the other classes.